orbit/tools/build/install-wine-macos.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Check if brew is installed
+if ! command -v brew >/dev/null 2>&1 ; then
+    echo "Homebrew is not installed. Please install Homebrew first. For instructions, see https://brew.sh/"
```

Comment:
probably not a big deal, but worth checking with Product if homebrew is a reasonable pre-req

Reply:
I will add a note how to install it without brew.